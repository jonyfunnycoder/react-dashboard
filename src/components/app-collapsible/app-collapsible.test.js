import React from 'react';
import renderer from "react-test-renderer";
import AppCollapsible from "./app-collapsible";

test('renders app collapsible correctly ', () => {
    const tree = renderer
        .create(
            <AppCollapsible>
                <h1>App Collapsible</h1>
            </AppCollapsible>
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});
