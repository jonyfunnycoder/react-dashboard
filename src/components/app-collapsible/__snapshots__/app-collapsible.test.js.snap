// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app collapsible correctly  1`] = `
<div
  className="app-card collapsible card"
>
  <h1>
    App Collapsible
  </h1>
</div>
`;
